### PR TITLE
#56734 When there is no font, the border should not appear. Display further guidance text.

### LINKED ISSUE #54527: Font Library:  A border is drawn when there are no fonts available
### Description

If the currently active theme does not define fonts, an unnatural border will be displayed.

actual

I would suggest either of the following approaches:

### Just hide the border

proposal_1

### Show some text

proposal_2

### Step-by-step reproduction instructions

- Activate Emptyempty theme.
- Open the Site Editor.
- Global Styles > Typography

### Screenshots, screen recording, code snippet

_No response_

### Environment info

_No response_

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

### PR DESCRIPTION
**Pull request by inc2734**
## What?

#56734 When there is no font, the border should not appear. Display further guidance text.

## Why?

Unnatural border have been displayed.

## How?

Ensure that borders are not displayed.
If custom fonts and theme fonts are missing, a message is displayed.

## Testing Instructions

1. Open site editor.
2. Open global style panel.
3. Open typography panel.
4. Remove font familes.

## Screenshots or screencast 
Before
image

After
image

### CODE REVIEW

**Comment by t-hamano** [comment]
This PR fixes #54527.

**Review by matiasbenedetto — COMMENTED** [review]
Hi @inc2734, thanks for the PR.
Technically, it looks good to me :+1: 

I want to confirm with @jasmussen about the text displayed when no fonts are activated.

**Comment by jasmussen** [comment]
Thanks for the ping. We can simplify the message a little, to just say "No fonts installed."

Add fonts from Google

File: packages/edit-site/src/components/global-styles/font-families.js [hunk 1]
```diff
@@ -51,14 +51,22 @@ function FontFamilies() {
...
+							<FontFamilyItem key={ font.slug } font={ font } />
+						) ) }
+						{ themeFonts.map( ( font ) => (
+							<FontFamilyItem key={ font.slug } font={ font } />
+						) ) }
+					</ItemGroup>
+				) : (
+					<>
+						{ __(
+							'Fonts are empty! Add some fonts and apply them to your site'
```
Comment (matiasbenedetto) [inline packages/edit-site/src/components/global-styles/font-families.js]: Could we update this text?

```suggestion
							'No fonts installed.'
```

**Review by matiasbenedetto — DISMISSED** [review]

**Review by t-hamano — APPROVED** [review]
LGTM 🚀 As a follow-up, we may need to display a similar message in a modal dialog when there is no font.

image

**Review by mikachan — APPROVED** [review]
This looks good to me too, thank you!

I've added some small commits:
- Fixed a linter error: https://github.com/WordPress/gutenberg/pull/56825/commits/74209cef724374913cfced8d639192dee8a1fb9e
- Slightly improved the readability of checking if there are fonts available: https://github.com/WordPress/gutenberg/pull/56825/commits/fa96f0f4317e316a947dfe4e11acd6d0100b6487

**Comment by t-hamano** [comment]
The readability improvements made in [fa96f0f](https://github.com/WordPress/gutenberg/pull/56825/commits/fa96f0f4317e316a947dfe4e11acd6d0100b6487) also make sense to me 👍

**Pull request merged**